docs/getting_started/configuration/poly_new/guide.rst

```diff
@@ -305,12 +305,12 @@ To enable ERT to read the simulation results, you'll need to use the :ref:`GEN_D
 
 .. code-block:: shell
 
-    GEN_DATA POLY_RES RESULT_FILE:poly_%d.out REPORT_STEPS:0 INPUT_FORMAT:ASCII
+    GEN_DATA POLY_RES RESULT_FILE:poly.out
 
 2. **Understanding the Arguments**:
 
 - **POLY_RES**: Name of this result set.
-- **RESULT_FILE:poly_%d.out**: Path to the file with the simulation results. 
+- **RESULT_FILE:poly.out**: Path to the file with the simulation results.
   The ``%d`` is always ``0``, but must be specified. (a requirement from earlier ERT versions).
 - **``REPORT_STEPS:0``**: Generally set to 0.
 - **``INPUT_FORMAT:ASCII``**: Specifies that the file is a text file (`ASCII` stands for "American Standard Code for Information Interchange").
```

Comment:
These do not make sense anymore, but they should be included in the general documentation for the keyword if they are not at the moment.
```suggestion
```